local_output/Tiltfile

```diff
@@ -0,0 +1,2 @@
+def local_output(command):
+  return str(local(command)).rstrip('\n')
```

Comment:
can you include support for the `quiet` keyword?

Reply:
What about just passing all the params to the local command? This way its semi future proof?

Reply:
Last I looked at Starlark, it didn't allow slurping kwargs. But worth taking another look.